src/randoop/sequence/Statement.java

```diff
@@ -67,11 +94,198 @@ public boolean equals(Object o) {
   @Override
   public int hashCode() {
```

Comment:
It's not your fault, but this is a bizarre and non-optimal hash function.  I suggest using instead
`return java.util.Objects.hash(statement, inputs)` and also simplify any other undesirable hash functions.